src/renderer/widgets/CreateWallet/ui/MultisigWallet/SelectSignatoriesThreshold.tsx

```diff
@@ -46,7 +46,9 @@ export const SelectSignatoriesThreshold = () => {
   const hasOwnedSignatory = !!ownedSignatoriesWallets && ownedSignatoriesWallets?.length > 0;
   const hasEnoughSignatories = signatories.length >= 2;
   const hasEmptySignatory = signatories.map(({ address }) => address).includes('');
-  const canSubmit = hasOwnedSignatory && hasEnoughSignatories && !multisigAlreadyExists && !hasEmptySignatory;
+  const isThresholdValid = threshold.value >= 2 && threshold.value <= signatories.length;
```

Comment:
Maybe add constant for 2 as `MIN_THRESHOLD` and use it in other places? In line 47 for example

Reply:
Good point. The 2 is set in stone in the pallets, so we can use it in several places. I'll change this.